boards/arm/faze/support/openocd.cfg

```diff
@@ -0,0 +1,39 @@
+#
+# Seagate FireCuda Gaming SSD (FaZe) board
+#
+
+# An external debug probe must be connected to the SWD port (4-pins J2 header).
+# Here we assume that a ST-LINK in-circuit debugger/programmer is used. You may
+# have to replace it with your own interface here.
+source [find interface/stlink.cfg]
```

Comment:
why not lpclink?

Reply:
Hi Andrei, It is because I am using a ST-LINK probe myself on this board. I'll do the change. Thanks.